go/server.go

```diff
@@ -250,6 +235,51 @@ func createPublicToken(c *gin.Context) {
 	})
 }
 
+func createLinkToken(c *gin.Context) {
+	linkToken, err := linkTokenCreate(nil)
+	if err != nil {
+		c.JSON(err.errorCode, gin.H{"error": err.error})
+	}
+	c.JSON(http.StatusOK, gin.H{"link_token": linkToken})
+}
+
+type httpError struct {
+	errorCode int
+	error     string
+}
+
+func (httpError *httpError) Error() string {
+	return httpError.error
+}
+
+// linkTokenCreate creates a link token using the specified parameters
+func linkTokenCreate(
+	paymentInitiation *plaid.PaymentInitiation,
+) (string, *httpError) {
+	countryCodes := strings.Split(PLAID_COUNTRY_CODES, ",")
+	products := strings.Split(PLAID_PRODUCTS, ",")
+	redirectURI := PLAID_REDIRECT_URI
+	configs := plaid.LinkTokenConfigs{
+		User: &plaid.LinkTokenUser{
+			ClientUserID: "user-id",
```

Comment:
nit: wondering if we should have an example / at least docs for developers here to specify a real looking user id (as opposed to the same user id every time)? could be as simple as having a fake email here (or essentially what we expect user id to look like)